packages/terra-site/src/examples/spacer/Index.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropsTable from 'terra-props-table';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-spacer/docs/README.md';
+import { version } from 'terra-spacer/package.json';
+
+// Component Source
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
+import SpacerSrc from '!raw-loader!terra-spacer/src/Spacer';
+
+// Example Files
+import Spacer from './SpacerExample';
+import SpacerResponsive from './SpacerResponsiveExample';
+
+const SpacerExamples = () => (
+  <div>
+    <div id="version">Version: {version}</div>
+    <Markdown id="readme" src={ReadMe} />
+    <PropsTable id="props" src={SpacerSrc} />
+    <br />
+    <h2 id="default">Spacer</h2>
```

Comment:
Lets update these example descriptions to describe what they are showing

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/e3ec8ffe0fc8945db3aadb3b1bdfc4547a3af3bc#diff-4a98913914f72896bd7b2c1918412f03L3